Day 1/DayOnePartOne.py

```diff
@@ -0,0 +1,38 @@
+import sys
+sys.path.insert(0, "C:/Users/Callipolis/Desktop/Programming/AoC2/Advent-of-Code-2015")
+from InputReader import InputReader
+
+input_reader = InputReader("input1.txt")
+raw_input = input_reader.get_raw_input()
+
+class FloorCounter:
```

Comment:
FloorCounter amugy tok jo nev. Este pont az jutott meg eszembe az agyban hogy atneveznem erre. Foleg ha a zarojel karakter hasonlitast kiveszed mint amit a masik kommentben irtam, akkor el is veszti az ertelmet a parent navigator

Reply:
nice ❤️❤️❤️